frontend/src/auth/components/PrivateRoute/PrivateRoute.tsx

```diff
@@ -2,17 +2,15 @@ import * as React from 'react';
 import { ROUTES } from 'global/constants';
 import { Redirect, Route, RouteProps } from 'react-router';
 import './styles.scss';
-import { checkAuthAndRefreshToken } from 'auth/utils/helpers';
 
 export const PrivateRoute: React.FC<RouteProps> = (props: RouteProps) => {
   const [isLoading, setIsLoading] = React.useState(true);
   const [isAuthenticated, setIsAuthenticated] = React.useState(false);
 
   React.useEffect(() => {
-    checkAuthAndRefreshToken().then((authenticated: boolean) => {
-      setIsAuthenticated(authenticated);
-      setIsLoading(false);
-    });
+    const accessToken = localStorage.getItem('token_access');
+    setIsAuthenticated(!!accessToken);
+    setIsLoading(false);
```

Comment:
The private route was refreshing the token too often, each time you'd access a private route. 
With this change we only check if an access token is set, and if it is, we consider the user logged in.

Reply:
Nice! That's perfect, but see my other comment regarding access token refresh issues.
I think I've found a good solution.